src/vdaf/xof.rs

```diff
@@ -91,12 +91,13 @@ impl<const SEED_SIZE: usize> Decode for Seed<SEED_SIZE> {
     }
 }
 
-/// A stream of bytes derived from a seed.
-pub trait SeedStream: Sized {
-    /// Fill `buf` with the next `buf.len()` bytes of output.
-    fn fill(&mut self, buf: &mut [u8]);
-
+/// Trait for deriving a vector of field elements.
+pub trait IntoVec: RngCore + Sized {
     /// Generate a finite field vector from the seed stream.
+    fn into_vec<F: FieldElement>(self, length: usize) -> Vec<F>;
+}
```

Comment:
This name is a little close to Rust's naming conventions for conversions. For example, slices have an `into_vec()` method that copies into an owned `Vec`. How about `IntoFieldVec`/`into_field_vec()` instead?

Reply:
Done.